src/components/pages/search-page/SearchPage.jsx

```diff
@@ -1,7 +1,47 @@
 import React from 'react'
+import LabelledInput from '../../labelled-input/LabelledInput'
+import './SearchPage.css'
+import Button from '../../button/Button'
 
 const SearchPage = () => {
-  return <div />
+  const centered = true
+  return (
+    <>
+      <form action="" className="search-page-form">
+        <LabelledInput
+          label="ID заказа"
+          labelClassName="search-page-form-fieldset"
+          centered={centered}
```

Comment:
```suggestion
          centered
```
Ты просто пишешь название пропа и все :)
Это равно
```jsx
centered={true}
```